docs/SESSION-OVER-SESSIONS.md

```
@@ -0,0 +1,508 @@
+);
+```
+
+WAL mode must be enabled (`PRAGMA journal_mode=WAL`) so the coordinator and session-handler can read concurrently without blocking.
+
+---
+
+## 3. New Server Modules
+
+All under `src/lib/modules/server/sos/`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)